1.16/Dockerfile.rhel7

```diff
@@ -0,0 +1,93 @@
+FROM rhscl/s2i-core-rhel7:1
+
+# RHSCL rh-nginx114 image.
+#
+# Volumes:
+#  * /var/opt/rh/rh-nginx114/log/nginx/ - Storage for logs
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV NAME=nginx \
+    NGINX_VERSION=1.14 \
```

Comment:
RHSCL 3.4 beta is already available, so we can definitely have 1.16 here already. Beta channel can be used below to make it work fine now, or we can simply keep the Dockerfile with main RHSCL channel as it is now and let the users who need to build it themselves with beta packages to change it (there are not many folks who do it I guess).